app/actions/route_mapping_delete.rb

```diff
@@ -22,6 +22,7 @@ def delete(route_mappings)
 
           route_mapping.destroy
           route_handler.update_route_information
+          CopilotHandler.new.unmap_route(route_mapping)
```

Comment:
The instantiation and calling style of the `CopilotHandler` and the `ProcessRouteHandler` should be consistent.

Both should either be instantiated and called in the same line, or different lines.

@selzoc && maryam